src/components/DataModificationPage.js

```diff
@@ -17,192 +17,203 @@ import TableRow from '@material-ui/core/TableRow';
+//         Edit </Typography></Button>
+
+//       table = <Table baseid="table-with-interactions" className="editable-data-table">
+//         <TableHead className="table-head">
+//           <TableRow>
+//             <TableCell numeric>Line Number</TableCell>
+//             <TableCell numeric>Units</TableCell>
+//             <TableCell numeric>Workers</TableCell>
+//             <TableCell numeric>Defects</TableCell>
+//             <TableCell numeric>Overtime</TableCell>
+//             <TableCell numeric>Quality Incidents</TableCell>
+//             <TableCell numeric>Downtime</TableCell>
+//             <TableCell>Notes</TableCell>
+//             <TableCell>Utilization</TableCell>
+
+//           </TableRow>
+//         </TableHead>
+
+//         <TableBody className="table-body">
+
+//           {/* The first row of data in table */}
+//           <TableRow>
+//             <TableCell> 1 </TableCell>
+//             <TableCell>{this.state.textFieldValue}</TableCell>
+//             <TableCell>25</TableCell>
+//             <TableCell>13</TableCell>
+//             <TableCell>33</TableCell>
+//             <TableCell>5</TableCell>
+//             <TableCell>21</TableCell>
+//             <TableCell>Running as scheduled</TableCell>
+//             <TableCell>1</TableCell>
+
+//           </TableRow>
+
+//           {/* Second row of data */}
+//           <TableRow>
+//             <TableCell>2</TableCell>
+//             <TableCell>230</TableCell>
+//             <TableCell>18</TableCell>
+//             <TableCell>9</TableCell>
+//             <TableCell>17</TableCell>
+//             <TableCell>11</TableCell>
+//             <TableCell>32</TableCell>
+//             <TableCell>Maintenance Required</TableCell>
+//             <TableCell>4</TableCell>
+//           </TableRow>
+
+//           {/* Third row of data */}
+//           <TableRow>
+//             <TableCell>3</TableCell>
+//             <TableCell>97</TableCell>
+//             <TableCell>14</TableCell>
+//             <TableCell>28</TableCell>
+//             <TableCell>50</TableCell>
+//             <TableCell>0</TableCell>
+//             <TableCell>19</TableCell>
+//             <TableCell>Running as scheduled</TableCell>
+//             <TableCell>1</TableCell>
+//           </TableRow>
+//         </TableBody>
+//       </Table>
+//     }
+
+//     return (
+//       <React.Fragment>
+
+//         <Grid container>
+//           <Grid item xs={12} style={{ marginTop: 10 }}>
+
+//             <Typography variant="display1" gutterBottom style={{ padding: 15, align: "center" }}>
+//               Line Data
+//             </Typography>
+
+//             <div>
+//               <Paper className="data-page">
+
+//                 {table}
+
+//               </Paper>
+//             </div>
+
+//             {button}
+
+//           </Grid>
+
+//         </Grid>
+//       </React.Fragment>
+//     );
+//   }
+// }
+
```

Comment:
Are we going to need the old code anymore?

Reply:
Nope, should be good to delete it.